src/EnergyPlus/General.cc

```diff
@@ -413,12 +404,7 @@ void DetermineDateTokens(EnergyPlusData &state,
 
     // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
     std::string CurrentString = String;
-    int Loop{};
     Array1D_string Fields(3);
-    int NumField1{};
-    int NumField2{};
-    int NumField3{};
-    bool errFlag{};
```

Comment:
[src/EnergyPlus/General.cc:418]:(style),[variableScope],The scope of the variable 'NumField1' can be reduced.
[src/EnergyPlus/General.cc:419]:(style),[variableScope],The scope of the variable 'NumField2' can be reduced.
[src/EnergyPlus/General.cc:420]:(style),[variableScope],The scope of the variable 'NumField3' can be reduced.
[src/EnergyPlus/General.cc:421]:(style),[variableScope],The scope of the variable 'errFlag' can be reduced.